src/unified/mod.rs

```diff
@@ -0,0 +1,100 @@
+use crate::auth::client::LoginResponse;
+use crate::auth::server::AuthMessage;
+use crate::chat::client::AchievementNotify;
+use crate::chat::server::{
+    AddFriendRequest as ChatAddFriendRequest, AddFriendResponse as ChatAddFriendResponse,
+    AddIgnore, RequestMinimumChatMode, RequestMinimumChatModePrivate, TeamInvite as ChatTeamInvite,
+    TeamInviteResponse, TeamLeave,
+};
+use crate::chat::{GeneralChatMessage, PrivateChatMessage};
+use crate::common::ServiceId;
+use crate::general::client::GeneralMessage;
+use crate::raknet::client::{
+    replica::{ReplicaConstruction, ReplicaSerialization},
+    ConnectedPong, ConnectionRequestAccepted,
+};
+use crate::raknet::server::{ConnectionRequest, InternalPing, NewIncomingConnection};
+use crate::world::client::{
+    AddFriendRequest, AddFriendResponse, BlueprintLoadItemResponse, BlueprintSaveResponse,
+    CharacterCreateResponse, CharacterDeleteResponse, CharacterListResponse, ChatModerationString,
+    CreateCharacter, FriendUpdateNotify, GetFriendsListResponse, GetIgnoreListResponse,
+    LoadStaticZone, MinimumChatModeResponse, MinimumChatModeResponsePrivate, TeamInvite,
+    TransferToWorld, UpdateFreeTrialStatus,
+};
+use crate::world::gm::client::SubjectGameMessage;
+use crate::world::server::WorldMessage;
+use endio::{Deserialize, Serialize};
+use lu_packets_derive::MessageFromVariants;
+
+#[derive(Debug, Deserialize, Serialize, PartialEq, MessageFromVariants)]
+#[non_exhaustive]
+#[repr(u8)]
+pub enum Message {
+    InternalPing(InternalPing) = 0,
```

Comment:
File uses spaces instead of tabs for indentation

Reply:
I remember specifically making sure to use tabs, it must've auto-formatted again without me noticing :sweat_smile: